data/json/items/classes/magazine.json

```diff
@@ -12,6 +12,7 @@
     "color": "ltgray",
     "rigid": false,
     "reliability": 6,
+    "capacity": 9999,
```

Comment:
> IRL you could make the belt as long as you want if you don't plan to move the gun around.

No, because your belt would become tangled if it's too long or require other materials to support it in coils or similar.

> Alternatively, belt could simply be set to become non-wearable if it is too encumbering.

I don't want to special case the code to support non-sensible lengths of belts

> How much capacity is realistic then?

100-500 rounds dependent upon the ammo, preferably dependent upon IRL examples.